app/views/pets/index.html.erb

```diff
@@ -1,13 +1,32 @@
-<%= link_to "Pets", "/pets/"%> <%= link_to "Shelters", "/shelters/"%>
+<div class="container">
+  <div class="row">
+    <div class="col-6 left">
+      <%= link_to "Pets", "/pets/"%> 
+    </div>
+    <div class="col-6 left">
+      <%= link_to "Shelters", "/shelters/"%> 
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6 center">
+      <h1>Pets</h1>
+    </div>
+  </div>
+</div>
+<div class="container">
+  <div class="row">
+    <div class="col-6 center">
+      <% @pets.each do |pet| %>
+        <h2><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></h2>
+        <img src=<%= "#{pet.img_url}" %> alt=<%= "#{pet.id}" %> height="100" width="100">
+        <p>Approximate Age: <%= pet.age %></p>
+        <p>Sex: <%= pet.sex %></p>
+        <p>Shelter: <%= link_to "#{pet.shelter.name}", "/shelters/#{pet.shelter.id}" %></p>
+        <%= link_to "Edit Pet", "/pets/#{pet.id}/edit"%>
+        <%= button_to "Delete Pet", "/pets/#{pet.id}", method: :delete %>
+      <% end %>
```

Comment:
Would like to find out in the future how to iterate into grid better. Struggled with that.